projects/libvpx/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -eu
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Build libvpx
+build_dir=$WORK/build
+mkdir -p ${build_dir}
+pushd ${build_dir}
+
+export CFLAGS="$CFLAGS -fno-sanitize=vptr"
+export CXXFLAGS="$CXXFLAGS -fno-sanitize=vptr"
```

Comment:
I thought instances of this in libwebm were gone, but you can also --disable-webm-io since all that's required here is ivfdec.c

Reply:
These flags were borrowed from ffmpeg/build.sh. Will remove these.

Reply:
Made the required changes in latest commit 3171ccac249ecd4f0877ed8ec9cdad00ec1e2e7b